.github/workflows/decision.yml

```diff
@@ -82,13 +83,7 @@ jobs:
           echo "subject=$subject" >> $GITHUB_OUTPUT
 
       - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v6
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          commit-message: 'Add documentation for decision: ${{ github.event.issue.title }}'
-          title: 'Automated PR ${{ github.event.issue.title }}'
-          labels: 'documentation,decision'
-          body: 'This PR was automatically created to document ${{ github.event.issue.number }}.'
-          base: 'main'
-          branch: 'docs/${{ github.event.issue.number }}_decision-${{ steps.new_branch.outputs.subject }}'
-          draft: true
+        run: |
+          gh pr create --base main --head 'docs/${ISSUE_NUMBER}_decision-${{ steps.new_branch.outputs.subject }}' --draft --label decision --label documentation --title "Add documentation for decision: ${ISSUE_TITLE}" --body "Created by Github action from ${ISSUE_NUMBER}"
```

Comment:
Needs double quotes instead of single quotes or else the shell will interpret $ literally

Reply:
Nice catch 😎